token/core/common/serializer.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package common
+
+import "github.com/hyperledger-labs/fabric-token-sdk/token/driver"
+
+type Serializer struct {
+}
+
+func (s Serializer) MarshalTokenRequestToSign(request *driver.TokenRequest, meta *driver.TokenRequestMetadata) ([]byte, error) {
```

Comment:
We could remove the metadata from the signature (it is the only implementation), unless other future implementations are supposed to use it.